Target-MVD-Mathias/Networking/Services/UserAPI.swift

```diff
@@ -74,6 +74,27 @@ class UserAPI {
       failure(error)
     }
   }
+  
+  class func signup(name: String, email: String, password: String, gender: String, success: @escaping (_ responseObject: [String: Any]) -> Void, failure: @escaping (_ error: Error) -> Void) {
+    let parameters = [
```

Comment:
If the swift-base is using 2 then all files are using this setting. You can change your XCode settings to use 2 spaces.